user/caribou/module/src/dSiPMEventConverter.cc

```diff
@@ -11,50 +11,72 @@ namespace {
       dSiPMEvent2StdEventConverter>(dSiPMEvent2StdEventConverter::m_id_factory);
 }
 
-bool dSiPMEvent2StdEventConverter::m_configured(0);
-bool dSiPMEvent2StdEventConverter::m_zeroSupp(1);
-bool dSiPMEvent2StdEventConverter::m_checkValid(0);
-uint64_t dSiPMEvent2StdEventConverter::m_trigger(0);
-uint64_t dSiPMEvent2StdEventConverter::m_frame(0);
-double dSiPMEvent2StdEventConverter::m_fine_ts_effective_bits[4];
+std::vector<bool> dSiPMEvent2StdEventConverter::m_configured({});
```

Comment:
Probably it would be cleaner to bake them all into a `struct` or `class`.

Reply:
Yup, looks significantly better now.